### DISCUSSION TITLE
Sigma File Manager v1.6.0

### DISCUSSION THREAD

**Discussion by aleksey-hoffman** [aleksey-hoffman/sigma-file-manager#200] [Announcements]
## Download

You can install the app in one of these ways:

<table>
  <thead>
    <tr>
      <th>Operating system</th>
      <th>Download link (latest version)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Windows (via winget)</td>
      <td width="320px">
        <code>winget install sigma --source winget</code>
      </td>
    </tr>
    <tr>
      <td>Windows (via Microsoft store)</td>
      <td>
        <a target="_blank" href="https://www.microsoft.com/store/productId/9N7W62ND0NF9">
          <img src="https://github.com/aleksey-hoffman/sigma-file-manager/raw/main/.github/media/get-from-microsoft.png" width="140px">
        </a>
        <br>⌛v1.6.0 is under review
      </td>
    </tr>
    <tr>
      <td>Windows (unsigned installable)</td>
      <td>
        <a target="_blank" href="https://github.com/aleksey-hoffman/sigma-file-manager/releases/download/v1.6.0/Sigma-File-Manager-1.6.0-Windows-Setup.exe">
          Installable file (.exe)
        </a>
      </td>
    </tr>
    <tr>
      <td>Debian (Ubuntu) portable app | experimental</td>
      <td>
        <a target="_blank" href="https://github.com/aleksey-hoffman/sigma-file-manager/releases/download/v1.6.0/Sigma-File-Manager-1.6.0-Linux-Debian.AppImage">
          AppImage
        </a>
      </td>
    </tr>
    <tr>
      <td>If there's no compiled binary for your Linux dist, you can build the app from the source code yourself, using <code>Node v15.14.0</code>
      </td>
      <td>
        <code>npm run build</code>
      </td>
    </tr>
  </tbody>
</table>

## Info
This app is primarily developed for Windows because it has the largest user base. The Linux builds get a lot less development time. They are less stable and have more issues. When this project has enough supporters, I will be able to spend more time on Linux builds.

#### Windows | Installation notes

It's recommended to install the app via the Microsoft store.
If you decide to install the `Installable file (.exe)` instead, you might see a security warning during installation. If you see the warning prompt, click **"More Info"** and then **"Run Anyway"**. 

#### Ubuntu 22.04 LTS | Installation notes

<a target="_blank" href="https://itsfoss.com/cant-run-appimage-ubuntu/">
  Running AppImage applications in Ubuntu 22.04 LTS
</a>

# Release Notes

- 1 new major feature
- 6 new features
- 11 improvements
- 13 fixes

## New major features
- **Localization:** Added ability to change UI language. Added 9 most popular languages.
  >**Note for contributors**: Detailed instructions for adding/editing a language are available at https://github.com/aleksey-hoffman/sigma-file-manager/discussions/196.

## New minor features
- **UI design:** Added option to make toolbars transparent;
- **Fonts:** Added option to change font;
- **Tabs:** Added option to display navigator tab preview;
- **Tabs:** Added option to display storage indicator on navigator tab;
- **Tabs:** Added option to change navigator tab width;
- **Date/time format:** Added option to choose date / time regional format;

## Improvements
- **Navigator:** Improved user experience for file management tasks such as copying, moving, creating, and renaming files. Now files will be scrolled into view and selected automatically;
- **Navigator:** The copy / move operations will now be disabled when any modal window is opened to avoid accidental file transfer actions;
- **Drag & drop:** The `.url` link-files are now treated as files;
- **UI design:** Improved and unified design for drive cards on both home page and nav panel;
- **Home banner:** Added ability to easily set previous home media from the home page via switch button (by pressing `Alt + Click`);
- **Search:** Increase max possible search scan depth from 10 to 15;
- **Search:** Improved "Recent items" list design;
- **Search:** Improved list item design to fit in more text;
- **Navigator:** Dir items will now only get highlighted when selecting dir item range, not just when pressing `Shift`;
- **New guide:** Added video downloading guide.
- **Other improvements:** Various small improvements;

## Major fixes
- **App:** Fixed the issue with the app not exiting properly;
- **Workspaces:** Fixed broken functionality and improved feature in general;
- **QuickView:** Fixed the issue with QuickView not working in production build;
- **QuickView:** Fixed issues with media size and background colors:
  - Fixed media size issue on Win 11 (scrollbars and videos jumps);
  - Fixed text files showing white text on white background on dark theme;
- **Search:** Fixed 'end of file' error during interrupted search data scan;
- **Navigator:** adding multiple items to protected / pinned list is now working properly;
- **Home banner:** Fixed animations, improved props adjustment UX;
- **Cards:** Fixed duplicate OneDrive cards issue;

## Minor fixes
- **Navigator:** Fixed the issue with navigator list stretching outside container when navigating directories with long address path;
- **Context menu:** Fixed the issue with active status for protected and pinned items not updating sometimes;
- **Styles:** Fixed broken styles in external program editor;
- **Styles:** Fixed visual filters not showing on the home page on first launch;

## Other changes
- **Home banner:** Added new default home banner image;
- **Search:** Search data compression setting is disabled by default, to avoid potential 'end of file' error;
- **Home banner:** Changed default overlay type to "mask-overlay";
- **Tabs:** Use traditional horizontal tab layout by default;

## Development
- **Dependencies:** Updated code dependencies.
- **Code refactor:** Various code refactoring.

<hr /><em>This discussion was created from the release <a href='https://github.com/aleksey-hoffman/sigma-file-manager/releases/tag/v1.6.0'>Sigma File Manager v1.6.0</a>.</em>

**Comment by Ansh4011** [comment]
FOMO is real. Belated congrats on the new release mate. 🍀

**Reply by aleksey-hoffman** [reply]
Hey Ansh :) Thanks
Check this out: https://github.com/aleksey-hoffman/sigma-file-manager/discussions/226

**Reply by Ansh4011** [reply]
> Hey Ansh :) Thanks Check this out: #226

Omg yes, that's super exciting bro! I'll be eagerly waiting for it to be out, SFM will be even snappier then.

**Reply by aleksey-hoffman** [reply]
@Ansh4011 thanks! Btw, I will soon start streaming on Twitch and record videos for YouTube, showing the process of building it :)
I think I will start in about 2-3 weeks. Just need to set up everything

**Reply by Ansh4011** [reply]
@aleksey-hoffman That's interesting mate, I use youtube so I'll definitely be watching you out there. What kind of videos do you plan to upload on youtube?

**Reply by aleksey-hoffman** [reply]
@Ansh4011 I'll be doing all kinds of videos, some educational, some funny meme code / design videos, do some chill / fun / answer questions streams :)

**Reply by Ansh4011** [reply]
@aleksey-hoffman that falls in my zone of interest xD, lmk when you start with it!
